quinn-proto/src/connection/paths.rs

```diff
@@ -19,23 +22,27 @@ impl PathData {
         remote: SocketAddr,
         initial_rtt: Duration,
         congestion: Box<dyn congestion::Controller>,
+        now: Instant,
     ) -> Self {
         PathData {
             remote,
             rtt: RttEstimator::new(initial_rtt),
             sending_ecn: true,
+            pacing: Pacer::new(congestion.window(), now),
             congestion,
             challenge: None,
             challenge_pending: false,
         }
     }
 
-    pub fn from_previous(remote: SocketAddr, prev: &PathData) -> Self {
+    pub fn from_previous(remote: SocketAddr, prev: &PathData, now: Instant) -> Self {
+        let congestion = prev.congestion.clone_box();
         PathData {
             remote,
             rtt: prev.rtt,
-            congestion: prev.congestion.clone_box(),
+            pacing: Pacer::new(congestion.window(), now),
```

Comment:
We're required to use the *initial* congestion window for the pacer capacity; this uses the current window. A simple solution would be to extend the `congestion::Controller` trait with a `fn initial_window(&self) -> u64`.

Reply:
It looks like this still needs to be addressed.

Reply:
I don’t have write access to the source branch anymore, so I would need to close this PR and open it with a different source branch.  Is that okay?